focus-gradle-plugin/src/main/kotlin/com/dropbox/focus/CreateFocusSettingsTask.kt

```diff
@@ -71,7 +101,11 @@ public abstract class CreateFocusSettingsTask : DefaultTask() {
     public operator fun invoke(subExtension: FocusSubExtension): CreateFocusSettingsTask.() -> Unit = {
       group = FOCUS_TASK_GROUP
       settingsFile.set(subExtension.focusSettingsFile)
-      notCompatibleWithConfigurationCache("This reads configurations from the project at action-time.")
+      modulesToDirMapFile.set(subExtension.moduleToDirMapFile)
+      projectPath.set(project.path)
+      // Collecting dependencies info in the configuration phase.
+      // Gradle disallows accessing project in execution phase.
+      projectDeps.set(project.collectDependencies())
```

Comment:
Does this end up forcing dependency resolution at configuration time? When I originally wrote this it was explicitly excluded from configuration caching and deferred dependency resolution to action-time to avoid exploding the configuration phase of a project due to dependency resolution.

Reply:
Well, yeah. I can't figure out how to support configuration cache and resolve dependencies at execution phase.
I tried accessing `ConfigurationContainer` but for some reason it's empty.

To reproduce this issue I add gradle.properties:
```
org.gradle.configuration-cache=true
org.gradle.configuration-cache.problems=warn
```

I reverted configuration cache changes.